styles/global.css

```diff
@@ -0,0 +1,121 @@
+:root {
+    --color-blue: #7388A0;
+    --color-darkblue: #313640;
+    --color-lightbeige: #FAF2E6;
+    --color-darkbeige: #DAA46D;
+    --color-brown: #9C4F20;
+    --color-darkgreen: #766621;
+    --font-size-text: 1.2rem;
+    --color-text: white;
+    font-family: "Roboto";
+    /*To get this font on each page you need to insert this into head of every html page.
+        <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+    */
+}
+
+div {
+    box-sizing: border-box;
+}
+
+html {
+    min-height: 100dvh;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.header {
+    position: fixed;
+    background-color: var(--color-lightbeige);
+    height: 10vh;
+    min-width: 100vw;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    -webkit-box-shadow: 0px 1px 1px 0px var(--color-lightbeige);
+    -moz-box-shadow: 0px 1px 1px 0px var(--color-lightbeige);
+    box-shadow: 0px 1px 5px 0px var(--color-lightbeige)
+}
+
+.header-logotype {
+    position: absolute;
+    height: 100%;
+    left: 8vw;
+}
```

Comment:
Perhaps, with the logotype change the color to one of the palette colors, either the original image or with a css filter.

you can use this website, but the colors may not be perfect:
https://isotropic.co/tool/hex-color-to-css-filter/